src/main/java/Game.java

```diff
@@ -233,4 +240,20 @@ private static String oppositeDirection(String d) {
 		return "";
 	}
 
+	//deliver message to all players
+	private static void deliverMessage(Instruction command, String senderName){
+		String message = createMessage(command.getItems().toString(), senderName);
+		for(PlayerThread player: users){
+			if(!player.getPlayerName().equals(senderName)){
+				player.output(message);
+			}
+		}
+	}
+
+	private static String createMessage(String temp, String name){
```

Comment:
very unimportant but temp probably isn't the best variable name

Reply:
Don't know how I overlooked that 🥇